source/renderer/app/stores/SidebarStore.spec.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import BigNumber from 'bignumber.js';
+
+import type { Api } from '../api/index';
+import type { ActionsMap } from '../actions/index';
+import type {
+  WalletSortByOptions,
+  WalletSortOrderOptions,
+} from '../types/sidebarTypes';
+import { WalletSortBy, WalletSortOrder } from '../types/sidebarTypes';
+
+import SidebarStore from './SidebarStore';
+
+describe('discreetWalletAmount replacer', () => {
+  const api: Api = ({
+    ada: jest.fn(),
+    localStorage: jest.fn(),
+  }: any);
+
+  const actions: ActionsMap = (jest.fn(): any);
+
+  function setupStore({
+    sortBy,
+    sortOrder,
+    wallets,
+  }: {
+    sortBy: WalletSortByOptions,
+    sortOrder: WalletSortOrderOptions,
+    wallets: Array<{ id: string, name: string, amount: BigNumber }>,
+  }) {
+    const sidebarStore = new SidebarStore(api, actions);
+    sidebarStore.walletSortConfig.sortBy = sortBy;
+    sidebarStore.walletSortConfig.sortOrder = sortOrder;
+
+    return sidebarStore._sortWallets((wallets: any));
```

Comment:
If we really want store integration tests then this would not be enough because we would have to use the public store API, not private methods that might never be called correctly using the public api